dev/ci/scripts/utils/gitlab/launch_gitlab_runner.sh

```diff
@@ -14,8 +14,7 @@ set -e
 #########################################################################
 
 # Set the HOMEgfs_ variable to the root directory of the global workflow
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-HOMEgfs_="$("${SCRIPT_DIR}/../find_homegfs.py")"
+HOMEgfs_="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../../../.." && pwd)"
```

Comment:
This works, but is kludge-y and will be a hard to find bug if the `gitlab` directory has to move.  Maybe instead of dropping the script, `find_homegfs.py` should be converted to a small Python wrapper around `find_upward`?

Reply:
It is, but then one has to know the location of the `find_homegfs.py` script.  So, no matter what, there is some amount of `../` stuff involved.

Reply:
Fair enough.  Something to ponder for a later date.

Reply:
Yea the whole point was to get ride of having to count ..'s that what I had in my version by design.

Reply:
A suggestion for a followup PR:

```
HOMEgfs_="$(git rev-parse --show-toplevel)"
```